### ISSUE TITLE
Error while running 

### ISSUE THREAD

**Issue by R34LUS3R** [subfinder/subfinder#177]
### Subfinder was working pretty fine about 3 days ago now I am getting this error

Note: I have tried it on 3 different instances and I am having this error on every single on of them.
Also removed and reinstall via go get and docker but same error.

$ uname -a
```Linux username 5.2.0-kali3-cloud-amd64 #1 SMP Debian 5.2.17-1kali1 (2019-09-27) x86_64 GNU/Linux```

$ go version
```go version go1.12.10 linux/amd64```

<details><summary><code>go env</code> Output</summary><br><pre>
$ go env
GOARCH="amd64"
GOBIN=""
GOCACHE="/root/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/root/go"
GOPROXY=""
GORACE=""
GOROOT="/usr/lib/go-1.12"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go-1.12/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build477441473=/tmp/go-build -gno-record-gcc-switches"
</pre></details>

```
Running enumeration on testing.com

dnsdb: Unexpected return status 404
2019/10/16 07:08:08 panic while running job: runtime error: index out of range
panic: runtime error: index out of range [recovered]
        panic: interface conversion: interface {} is runtime.errorString, not string

goroutine 91 [running]:
github.com/subfinder/subfinder/libsubfinder/helper.(*Pool).subworker.func1(0xc000110410)
        /root/go/src/github.com/subfinder/subfinder/libsubfinder/helper/pool.go:61 +0x1c9
panic(0x75e8a0, 0xa96a00)
        /usr/lib/go-1.12/src/runtime/panic.go:522 +0x1b5
github.com/subfinder/subfinder/libsubfinder/sources/dnsdumpster.Query(0xc000114460, 0x2, 0x2, 0x6c7101, 0xaa0101)
        /root/go/src/github.com/subfinder/subfinder/libsubfinder/sources/dnsdumpster/dnsdumpster.go:61 +0x14a6
github.com/subfinder/subfinder/libsubfinder/helper.(*Pool).subworker(0xc000080e60, 0xc000110410)
        /root/go/src/github.com/subfinder/subfinder/libsubfinder/helper/pool.go:64 +0x6e
github.com/subfinder/subfinder/libsubfinder/helper.(*Pool).worker(0xc000080e60, 0x18)
        /root/go/src/github.com/subfinder/subfinder/libsubfinder/helper/pool.go:78 +0xb3
created by github.com/subfinder/subfinder/libsubfinder/helper.(*Pool).Run
        /root/go/src/github.com/subfinder/subfinder/libsubfinder/helper/pool.go:181 +0x7d
```

Thanks if you could help me out.

**Comment by VictorAlonsoCM** [comment]
I have the same issue

**Comment by MRSM1L3S** [comment]
Same

**Comment by VictorAlonsoCM** [comment]
This works for me: **--exclude-sources dnsdumpster**

**Comment by MRSM1L3S** [comment]
> This works for me: **--exclude-sources dnsdumpster**

Agreed, I also applied PR #178 as I was running into the errors listed there _after_ applying your recommendation. Anyways, thanks for sharing this important tip, take care!

**Comment by realuser07** [comment]
Yup, It's working for me also with this flag: ```--exclude-sources dnsdumpster```
Thanks @VictorAlonsoCM

**Comment by ZeroDot1** [comment]
I have the same issue

**Comment by Ice3man543** [comment]
I'll take a look into it just now

**Comment by Ice3man543** [comment]
Hi, this should be fixed with PR #179.

Within a month, a new version of subfinder is coming with a lot of performance improvements, better code, etc. It's been in dev for months and is way better. Sadly, i've been stuck in some other stuff, hope to release it soon for the community to use and enjoy!

**Comment by Ice3man543** [comment]
It's been fixed! Let me know if anyone else has the same issue
